src/orange/OrangeTrackView.hh

```diff
@@ -263,9 +263,8 @@ OrangeTrackView::operator=(Initializer_t const& init)
         if (daughter_id)
         {
             auto const& daughter = params_.daughters[daughter_id];
-            auto const& trans = params_.translations[daughter.translation_id];
-            TranslatorDown td(trans);
-            local.pos = td(local.pos);
+            Translation trans{params_.translations[daughter.translation_id]};
```

Comment:
These will eventually be Transformation objects, correct?

Reply:
I think this will eventually be called through a type-deleted functor like SurfacesAction.